upgrade/v1beta1/upgrade.go

```diff
@@ -192,15 +210,74 @@ func upgradeNodes(namespace string, lhClient *lhclientset.Clientset) error {
 			return errors.Wrap(err, "failed to copy node Status.DiskStatus")
 		}
 
-		_, err = lhClient.LonghornV1beta2().Nodes(namespace).UpdateStatus(context.TODO(), &new, metav1.UpdateOptions{})
+		obj, err := lhClient.LonghornV1beta2().Nodes(namespace).UpdateStatus(context.TODO(), &new, metav1.UpdateOptions{})
 		if err != nil {
 			return errors.Wrapf(err, "failed to update for %v status %v", old.Kind, old.Name)
 		}
+		if err := addCRDVersionLabelToNode(obj, namespace, lhClient); err != nil {
+			return errors.Wrapf(err, "failed to update for %v %v", old.Kind, old.Name)
+		}
 	}
 	logrus.Info("Finished upgrading nodes")
 	return nil
 }
 
+func addCRDVersionLabelToVolume(obj *longhorn.Volume, namespace string, lhClient *lhclientset.Clientset) error {
```

Comment:
Maybe you use use a single function like [this](https://github.com/longhorn/longhorn-manager/blob/7bd89abdbae3c406ec56094364cf6c2da1d36d9d/datastore/longhorn.go#L316) rather than one labeling function for each CRD.